aws/config.go

```diff
@@ -206,6 +207,7 @@ type AWSClient struct {
 	athenaconn            *athena.Athena
 	dxconn                *directconnect.DirectConnect
 	mediastoreconn        *mediastore.MediaStore
+	wsconn                *workspaces.WorkSpaces
```

Comment:
Minor nitpick: to prevent any confusion with other services, we should probably name this `workspacesconn`